db/install.xml

```diff
@@ -20,4 +20,4 @@
       </INDEXES>
     </TABLE>
   </TABLES>
-</XMLDB>
\ No newline at end of file
+</XMLDB>
```

Comment:
I am not sure why we need to add the newline (Moodle unfortunately creates this way)

Reply:
This is breaking the unit test on ubuntu servers, adding the newline fixes the test.